HACKING.adoc

```diff
@@ -229,6 +229,74 @@ opam switch create my-switch-name --empty
 opam pin add ocaml-variants.$VERSION+branch git+https://$REPO#branch
 ----
 
+=== Incremental builds with `opam`
+
+If you want to fix a bug that needs opam packages to reproduce, or want to try
+a new feature that you're still actively developing on opam packages, you will
+likely need a faster feedback loop than re-creating a new switch or recompiling
+every package each time you make a small change to your code.
+
+There are two mostly orthogonal steps you can take to accelerate this:
+- Use your locally built copy, and only run the install step
+- Prevent your changes from automatically triggering the rebuild of all packages
+
+For the first step, one solution is to use the `--assume-built` flag to
+`opam install` to skip the compilation and only run the installation instructions.
+This, however, requires that your working directory is in a state suitable for
+running `make install`. This means it should be configured to the correct prefix,
```

Comment:
"This means..." I was constantly asking myself, how I'm going to do that and it was not immediately apparent to me that the instructions below were achieving that, somehow this could be made more clear (maybe show the steps first and explain them after ?).